newIDE/app/yarn.lock

```diff
@@ -5154,13 +5154,6 @@
     "buffers" "~0.1.1"
     "chainsaw" "~0.1.0"
 
-"bindings@^1.5.0":
```

Comment:
Please do not include lockfiles in your commits.

Reply:
I don't know how to remove them from the commit with git